containers/src/Kokkos_DynRankView.hpp

```diff
@@ -1008,15 +1008,15 @@ class DynRankView : public ViewTraits<DataType, Properties...> {
   ~DynRankView() = default;
 
   KOKKOS_INLINE_FUNCTION
-  DynRankView() : m_track(), m_map(), m_rank() {}  // Default ctor
+  DynRankView() : m_map(), m_track(), m_rank() {}  // Default ctor
 
   KOKKOS_INLINE_FUNCTION
   DynRankView(const DynRankView& rhs)
-      : m_track(rhs.m_track), m_map(rhs.m_map), m_rank(rhs.m_rank) {}
+      : m_map(rhs.m_map), m_track(rhs.m_track), m_rank(rhs.m_rank) {}
```

Comment:
Obviously you didn't write this copy constructor originally, but @crtrott @dhollman - why not just default the copy constructor here?

Reply:
resolved with 66564ee.

Interesting note: the default constructor didn't work when I made it defaulted...so I left it as it was

Reply:
@jeffmiles63 If you define any constructor (e.g., the copy constructor), even as `=default`, then the default constructor is not declared.  You would have to do `KOKKOS_INLINE_FUNCTION DynRankView() = default;` to get it back.